common/model-views.cpp

```diff
@@ -2917,6 +2921,102 @@ namespace rs2
                         }
                     }
                 }
+
+                // Draw IR reflectivity on depth frame
+                if (!viewer.skip_option(RS2_OPTION_ENABLE_IR_REFLECTIVITY))
+                {
+                    if (ds.supports(RS2_OPTION_ENABLE_IR_REFLECTIVITY) && (ds.get_option(RS2_OPTION_ENABLE_IR_REFLECTIVITY) == 1.0f))
+                    {
+                        // Add depth sample to the history
+                        viewer.ref.add_input(val, x, y);
+                        _reflectivity_data_collected = true;
+
+                        float noise_est = ds.get_option(RS2_OPTION_NOISE_ESTIMATION);
```

Comment:
Remind me -- the noise estimation is now shown in the Viewer? With the temperatures?

Reply:
Yes, I will add it to the hidden options for the viewer+ DQT since nobody asked for it.